syntax = "proto3";

package Text_analys;


service TextAnalysService  {
  rpc getResult (SettingsTextPB) returns (ResultParsingPB) {}
}


message SettingsTextPB {
  string text = 1;
}

//water 1..10
//hard_reading 1..10

message ResultParsingPB {

  enum Mood {
    sad = 0;
    happy = 1;
    lovely = 2;
    terrible = 3;
    boring = 4;
  }

  int32 water_value = 1;
  Mood mood = 2;
  int32 hard_reading = 3;
}
